style.css

```diff
@@ -0,0 +1,87 @@
+*{
+    padding: 0rem;
+    margin: 0rem;
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+}
+body{
+    display: flex; 
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    padding: .8rem;
+    top:0;
+    width: 100%;
+}
+.font__a{
+    text-decoration: none;
+    color: black;
+    font-family: 'Montserrat', sans-serif;
+    cursor: pointer;
+}
+.header__a{
+    font-family: 'Montserrat', sans-serif;
+    font-weight: bold;
+    font-size: 38px;
+    padding-top: .5rem;
+}
+
+.header div button{
+    font-size: 30px;
+    margin: .2rem;
+    border: .1rem solid #72C1B2;
+    background-color: white;
+    padding: .9rem;
+    border-radius: 2rem;
+    font-weight:initial;    
+}
+.header div button:hover{
+    background-color: #72C1B2;
+}
+.body__div{
+    background-color:#88C8BC ;
+    height: 80px;
+    width: 100%;
+}
+.body__div h1 {
+    padding :20px;
+    color: white;
+    font-size:30px ;
+    font-weight: initial;
+    font-style: normal;
+}
+.bode__pic{
```

Comment:
Hey 👋
I think the main image problem is caused by a typo here: instead of `.bode__pic {` you should change it to `.body__pic {` because that's how it is written in index.html

Another issue I noticed is with the unsplash url:
`https://unsplash.com/photos/zwsHjakE_iI?utm_source=unsplash&utm_medium=referral&utm_content=creditShareLink`

That link is to an unsplash webpage, it's not a link to the image itself.  I think you want something like this link for your image instead:
`https://images.unsplash.com/photo-1522199755839-a2bacb67c546?ixlib=rb-1.2.1&auto=format&fit=crop&w=3452&q=80`

And then you might have to make the `.body__pic` taller than `height: 200px` to see it well.  Let me know if all of that works for you!

Reply:
thx Tom . its worked ✨👌

Reply:
Cool!